### PR TITLE
GHI-271 Added integration test to change owners

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- New EIP712 hashing method
- Change Threshold
- Add Owner
- Swap Owner
- Remove Owner

### CODE REVIEW

File: MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift [hunk 1]
```diff
@@ -116,38 +116,103 @@ class EncryptionServiceTests: XCTestCase {
...
+        let dataHashInput = oneline("561001057600080fd5b5060405161060a3803806106")
+        let dataHash = keccak(dataHashInput)
+        print(dataHashInput, dataHash)
+
+        let domainHashInput = oneline("""
+035aff83d86937d35b32e04f0ddc6ff469290eef2f1b692d8a815c89404d4749
+000000000000000000000000092CC1854399ADc38Dad4f846E369C40D0a40307
+""")
+        let domainHash = keccak(domainHashInput)
+        print(domainHashInput, domainHash)
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift:130]: please remove

File: MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift [hunk 2]
```diff
@@ -116,38 +116,103 @@ class EncryptionServiceTests: XCTestCase {
         XCTAssertEqual(rinkebySignedTx.value, RawTransactionFixture.rinkebyHash)
     }
 
+    // swiftlint:disable:next function_body_length
+    func test_transactionHash_byParts() throws {
+        let dataHashInput = oneline("561001057600080fd5b5060405161060a3803806106")
+        let dataHash = keccak(dataHashInput)
+        print(dataHashInput, dataHash)
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift:123]: please remove

**Review by sche — APPROVED** [review]

File: MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift (hunk 1 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift:130]: I'll keep it for test purposes

File: MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift (hunk 2 above)
Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletImplementations/EncryptionServiceTests.swift:123]: I'll keep it for test purposes

**Pull request merged**